test/unittest/pointertest.cpp

```diff
@@ -1,6 +1,7 @@
 // Tencent is pleased to support the open source community by making RapidJSON available.
 // 
 // Copyright (C) 2015 THL A29 Limited, a Tencent company, and Milo Yip.
+// Portions (C) Copyright IBM Corporation 2021
```

Comment:
I am not sure if this is legally OK for my company. Can it be removed?

Reply:
@miloyip I have removed it. It's a standard IBM thing, but we are fine if you are not comfortable with it.